### PR TITLE
Map view

### PR DESCRIPTION
**Pull request by fangsgiving**

### CODE REVIEW

**Comment by fangsgiving** [comment]
@jkradddic @tekiek edit view

**Comment by jkradddic** [comment]
@fangsgiving  Think you're missing an updated `urls.py` file. Also do you set the map_id anywhere?

**Comment by fangsgiving** [comment]
- Didn't add the urls.py page because @tekiek wanted to render the edit view on the same page as the gallery
- I'm getting the map_id from Mapper, which will be passed on click from the gallery view.

File: mapper/static/js/mapper_generate.js [hunk 1]
```diff
@@ -9,16 +9,34 @@ Mapper.generate = new function() {
...
 		Mapper.map_components.storageMarkerLayer = new L.geoJson().addTo(Mapper.map_components.map);
+
 		Mapper.map_components.storageMarkerLayer.options.pointToLayer = L.mapbox.marker.style;
 
 		Mapper.generate.addListeners();
 
 	}
 
+	this.loadMap = function() {
+			Mapper.map_components.activeMarkerLayer = new L.mapbox.featureLayer()
```
Comment (jkradddic) [inline mapper/static/js/mapper_generate.js]: You actually want to add all these points to the `Mapper.map_components.storageMarkerLayer`. The active layer is reserved for a single marker that's being created/edited.

**Pull request merged**